css/mainPC.css

```
@@ -0,0 +1,155 @@
+@font-face {
+    font-family: "Angst";
+    src: url(/constellations/fonts/angst/static/Angst-Normal.otf);
+}
+
+main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 100vw;
+    height: 100vh;
+
+    background-image: url('../mainPNG/background.svg');
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.logo {
+    width: 10vw;
+    height: 10vh;
+
+    top: 0;
+}
+
+.main-container {
+    display: flex;
+
+    flex-direction: column;
+    align-items: center;
+
+    height: 90%;
+    width: 95%;
+}
+
+.upper-container {
+    display: flex;
+
+    width: 100%;
+    height: 65%;
+}
+
+.upper-container .left-container {
+    display: flex;
+
+    flex-direction: column;
+
+    width: 11%;
+    height: 100%;
+}
+
+.buttons {
+    display: flex;
+
+    flex-direction: column;
+    justify-content: space-between;
+
+    height: 40%;
+
+    margin-top: 85%;
+}
+
+.button {
+    width: 60%;
+
+    cursor: pointer;
+    user-select: none;
+}
+
+.midle-container {
```

сменить имя селектора в соответсвии с конвенцией нейминга, например можно ознакомиться с методологией БЭМ
```suggestion
.home-page__constellation-wheel_middle-container

```